contrib/dpu_daemon/scripts/config.sh

```diff
@@ -0,0 +1,46 @@
+GH=${GH:-0}
+if [ ${GH} -eq 1 ]; then
+    GH_FOLD="::group::"
+    GH_UNFOLD="::endgroup::"
+fi
+export MULTI_BUILD=${MULTI_BUILD:-1}
+export WORK_DIR=${WORK_DIR:-/global/scratch/users/dpu/deploy}
+export BUILD_DIR="$WORK_DIR/build-$arch"
+export SCRIPT_DIR=$PWD
+
+export UCX_SRC="$WORK_DIR/ucx"
+export UCX_DIR="$BUILD_DIR/ucx"
+export UCX_URL='https://github.com/openucx/ucx.git'
+export UCX_BRANCH=${UCX_BRANCH:-v1.14.x}
+
+export UCC_SRC="$WORK_DIR/ucc"
+export UCC_DIR="$BUILD_DIR/ucc"
+export UCC_URL='https://github.com/B-a-S/ucc.git'
+export UCC_BRANCH=${UCC_BRANCH:-v0.4.x}
+
+export MPI_SRC="$WORK_DIR/ompi"
+export MPI_DIR="$BUILD_DIR/ompi"
+export MPI_URL='https://github.com/open-mpi/ompi.git'
+export MPI_BRANCH=${MPI_BRANCH:-v5.0.x}
+
+export OMB_SRC="$WORK_DIR/omb"
+export OMB_DIR="$BUILD_DIR/omb"
+export OMB_URL='https://github.com/paklui/osu-micro-benchmarks.git'
+export OMB_BRANCH=${OMB_BRANCH:-master}
+
+completed (){
+    if [ ${MULTI_BUILD} -eq 1 ]; then
+        touch ${WORK_DIR}/TMP/${1}
+    fi
+}
+
+waiting (){
```

Comment:
Maybe rename to something like `wait_for` (infinitive is usually used in function names)?

Reply:
Renamed